frontend/src/modules/AppWrapper.tsx

```diff
@@ -1,9 +1,9 @@
 import React, { Suspense } from "react";
-import { Spin } from "antd";
 import { useThemeSwitcher } from "react-css-theme-switcher";
 import ROUTES from "Src/constants/routes";
 import { IS_LOGGED_IN } from "Src/constants/auth";
 import { BrowserRouter, Route, Switch, Redirect } from "react-router-dom";
+import { CustomSpinner } from "./../components/Spiner";
```

Comment:
Can we import from: `Src/components/..`

Reply:
i didn't get this @jyash97 can u elaborate!